vulture/whitelists/typing_whitelist.py

```diff
@@ -0,0 +1,14 @@
+try:
+    import typing
+
+    typing.Any
+    typing.Callable
+    typing.Dict
+    typing.Iterable
+    typing.List
+    typing.Optional
+    typing.Set
+    typing.Tuple
+    typing.Union
+except ImportError:
```

Comment:
Some of the classes from https://docs.python.org/3/library/typing.html#classes-functions-and-decorators are missing - for example:
```
typing.AnyStr
typing.ClassVar
typing.NoReturn
```